security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,30 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-0-0"]
+== OpenShift Compliance Operator 1.0.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.0.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:1682[RHBA-2023:1682 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-1-0-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The Compliance Operator is now stable and the release channel is upgraded to `stable`. Future releases will follow link:https://semver.org/[Semantic Versioning]. To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
+
+[id="compliance-operator-1-0-0-bug-fixes"]
+=== Bug fixes
+
+// 4.13 only, commenting out for now
+// * Before this update, the `rhcos4-ensure-logrotate-activated` rule failed after auto-remediation was applied in {product-title} 4.13 environments. With this update, rules with auto-remediation available have a `PASS` result after the auto-remedations are applied. (link:https://issues.redhat.com/browse/OCPBUGS-10769[*OCPBUGS-10769*])
+
+* Before this update, the compliance_operator_compliance_scan_error_total metric had an ERROR label with a different value for each error message. With this update, the compliance_operator_compliance_scan_error_total metric does not increase in values. (link:https://issues.redhat.com/browse/OCPBUGS-1803[*OCPBUGS-1803*])
```

Comment:
compliance_operator_compliance_scan_error_total looks like it should be in back ticks. Am I missing something here?